spring-data-relational/src/test/java/org/springframework/data/relational/core/mapping/DefaultAggregatePathUnitTests.java

```diff
@@ -467,6 +467,20 @@ void getEffectiveIdColumnName() {
 		});
 	}
 
+	@Test // GH-1802
+	void dingens() {
```

Comment:
Dingens?

Reply:
Oops, sorry. I thought gave that thing a proper name 😢